packages/react/src/sub-nav/utils.ts

```diff
@@ -1,9 +1,7 @@
 export const localPaletteVars = {
 	linkHoverBg: '--nav-linkHoverBg',
-	bottomBar: '--nav-bottomBar',
```

Comment:
Sub nav never used `localPaletteVars.bottomBar` so going to remove this - I believe this was a copy/paste error.